README.md

```diff
@@ -26,23 +26,24 @@ at <https://www.zulip.org>.
 There are several places online where folks discuss Zulip.
 
 * The primary place is the
-  [Zulip development community Zulip server](https://zulip.readthedocs.io/en/latest/chat-zulip-org.html).
+  [Zulip development community Zulip server][czo-doc] at
+  chat.zulip.org.
 
 * For Google Summer of Code students and applicants, we have
 [a mailing list](https://groups.google.com/forum/#!forum/zulip-gsoc)
 for help, questions, and announcements.  But it's often simpler to
-visit chat.zulip.org instead.
-
-* We have
-[a public mailing list](https://groups.google.com/forum/#!forum/zulip-devel)
-that is currently pretty low traffic because most discussions happen
-in our public Zulip instance.  We use it to announce Zulip developer
-community gatherings and ask for feedback on major technical or design
-decisions.  It has several hundred subscribers, so you can use it to
-ask questions about features or possible bugs, but please don't use it
-ask for generic help getting started as a contributor (e.g. because
-you want to do Google Summer of Code).  The rest of this page covers
-how to get involved in the Zulip project in detail.
+[visit chat.zulip.org][czo-doc] instead.
+
+* We have a [public development discussion mailing list][zulip-devel],
+[zulip-devel][zulip-devel], which is currently pretty low traffic
+because most discussions happen in our public Zulip instance.  We use
+it to announce Zulip developer community gatherings and ask for
+feedback on major technical or design decisions.  It has several
+hundred subscribers, so you can use it to ask questions about features
+or possible bugs, but please don't use it ask for generic help getting
```

Comment:
we've been thinking about renaming it to zulip-devel-announce; we recently moved it to being moderated a while back.  So it might make sense to do a further change here on that front.